cli/src/init/write_toml.rs

```diff
@@ -0,0 +1,46 @@
+pub fn init_toml() -> &'static str {
+    r#"
+[package]
+name = "MOPRO_TEMPLATE_PROJECT_NAME"
+version = "0.1.0"
+edition = "2021"
+
+[lib]
+name = "mopro"
+crate-type = ["lib", "cdylib", "staticlib"]
+
+# Adapters for different proof systems
+[features]
+default = []
+
+[dependencies]
+mopro-wasm = { git = "https://github.com/zkmopro/mopro.git" }
+mopro-ffi = { git = "https://github.com/zkmopro/mopro.git", features = ["uniffi"] }
+thiserror = "2.0.12"
+anyhow = "1.0.99"
+
+# CIRCOM_DEPENDENCIES
+# HALO2_DEPENDENCIES
+# NOIR_DEPENDENCIES
+
+[build-dependencies]
+# CIRCOM_BUILD_DEPENDENCIES
+# HALO2_BUILD_DEPENDENCIES
+# NOIR_BUILD_DEPENDENCIES
+
+[dev-dependencies]
+mopro-ffi = { git = "https://github.com/zkmopro/mopro.git", features = ["uniffi-tests"] }
+
+# CIRCOM_DEV_DEPENDENCIES
+# HALO2_DEV_DEPENDENCIES
+# NOIR_DEV_DEPENDENCIES
+
+[patch.crates-io]
+rust-rapidsnark = { git = "https://github.com/ElusAegis/rust-rapidsnark" }
+witnesscalc-adapter = { git = "https://github.com/ElusAegis/witnesscalc_adapter" }
```

Comment:
does it work with `dylib`?
(the one which is published
https://crates.io/crates/rust-rapidsnark
https://crates.io/crates/witnesscalc-adapter